scenes/music_puzzle/musician.dialogue

```diff
@@ -0,0 +1,16 @@
+~ start
+Musician: Hello there, Storyweaver.
+Musician: Have you seen those stones all over this island?
+- What about them? They're just inert lumps of rock.
+	Musician: Oh, I wouldn't be so sure about that!
+- There's something unusual about them… it's like they have a certain energy.
+Musician: They're tuned to a major scale!
+	Legend has it that playing the right melody will open a door.
+=> demo_phrase_1
```

Comment:
Maybe we could rename this label to be related to the actual dialogue?

Reply:
I think I'll not make any further tweaks here because this is placeholder dialogue but I'm interested in your advice on how to structure dialogue in general :)